### PR TITLE
fix: outputs sum should ignore authority outputs

### LINKED ISSUE #28: Sum of outputs should ignore authority outputs
This is important because this sum is used as a parameter for the tx weight.

### PR DESCRIPTION
**Pull request by obiyankenobi**
Fixes #28

### CODE REVIEW

File: src/transaction.js [hunk 1]
```diff
@@ -408,6 +405,35 @@ const transaction = {
...
+   * }]
+   *
+   * @return {number} Sum of outputs
+   * @memberof Transaction
+   * @inner
+   */
+  getOutputsSum(outputs) {
+    let sumOutputs = 0;
+    for (let output of outputs) {
+      // XXX wallet.js has method isAuthorityOutput, but it expects an output
```
Comment (pedroferreira1) [inline src/transaction.js]: So maybe we should create a method `isTokenDataAuthority` that receives the token data and has only the line `return output.tokenData & TOKEN_AUTHORITY_MASK`.  This method would be used by both methods `isAuthorityOutput` and `getOutputsSum `

Comment (obiyankenobi) [reply src/transaction.js]: Done

File: src/transaction.js [hunk 2]
```diff
@@ -636,6 +658,20 @@ const transaction = {
   clearTransactionWeightConstants() {
     this._weightConstants = null;
   },
+
+  /*
+   * Verifies if output is an authority one checking with authority mask
+   *
+   * @param {Object} output Output object with 'token_data' key
```
[the remark this answers is not in the document]
Comment (obiyankenobi) [reply src/transaction.js]: Thanks

File: src/transaction.js [hunk 3]
```diff
@@ -636,6 +658,20 @@ const transaction = {
...
+   *
+   * @return {boolean} if output is authority
+   *
+   * @memberof Wallet
```
[the remark this answers is not in the document]
Comment (obiyankenobi) [reply src/transaction.js]: Thanks

File: src/transaction.js [hunk 4]
```diff
@@ -636,6 +658,20 @@ const transaction = {
   clearTransactionWeightConstants() {
     this._weightConstants = null;
   },
+
+  /*
+   * Verifies if output is an authority one checking with authority mask
```
[the remark this answers is not in the document]
Comment (obiyankenobi) [reply src/transaction.js]: Thanks

File: src/transaction.js [hunk 5]
```diff
@@ -636,6 +658,20 @@ const transaction = {
...
+   *
+   * @param {Object} output Output object with 'token_data' key
+   *
+   * @return {boolean} if output is authority
```
[the remark this answers is not in the document]
Comment (obiyankenobi) [reply src/transaction.js]: Thanks

**Pull request merged**